assets/css/_diversion_page.scss

```diff
@@ -92,6 +92,9 @@
   display: flex;
   justify-content: flex-end;
   align-items: center;
+  &.active-detour {
+    background-color: $color-lemon-100;
```

Comment:
Idea: Since I'm going to use the same color for the background of the active detours table, could/should we make this into a semantic color, named `$active-detour` or something like that?

Non-blocking.